addons/bus/static/src/css/async.scss

```diff
@@ -0,0 +1,39 @@
+.o_ready_job_counter {
+    margin-top: -0.8rem;
+    margin-right: 0;
+    margin-left: -0.6rem;
+    background: #00A09D;
+    color: white;
+    vertical-align: super;
+    font-size: 0.7em;
+}
+
+.o_async_jobs {
+    width: 250px;
+    display: flex;
+    flex-direction: column;
+
+    .o_async_job {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 0.3em 0;
+
+        &:not(:last-child) {
+            border-bottom: 1px solid gray('400');
+        }
+
+        .o_async_job_state {
+            flex-basis: 0;
+            padding: 0 0.4em;
+        }
+
+        &.o_async_job_succeeded, &.o_async_job_failed {
+            cursor: pointer;
+        }
+    }
+
+    .o_async_jobs_empty {
+        padding: 3px 20px;
+    }
+}
```

Comment:
- Name the file similarly to the widget JS file, e.g. `bus/static/src/widgets/async_job_systray_menu/async_job_systray_menu.scss` (yes, JS and SCSS in same folder)
- Avoid rule hierarchy in SCSS. I know this is a SCSS feature that maps to another one in CSS (CSS specificity), and we use that a lot in our CSS of Odoo, but the truth is that this is a pitfall for having hard-to-maintain CSS. Keep SCSS rules as flat as possible. You can take inspiration with file `mail/static/src/components/messaging_menu/messaging_menu.scss`.